app/Config/Constants.php

```diff
@@ -25,6 +25,16 @@
  */
 defined('COMPOSER_PATH') || define('COMPOSER_PATH', ROOTPATH . 'vendor/autoload.php');
 
+/*
+ | --------------------------------------------------------------------
+ | Environment File
+ | --------------------------------------------------------------------
+ |
+ | This define allows you to control the name of the environment file so that 
+ | you can manipulate more than one environment attached to a single core system.
+ */
+defined('ENVIRONMENT_FILE') || define('ENVIRONMENT_FILE', '.env');
```

Comment:
```suggestion
 | This allows you to control the name of the env file so that you can handle 
 | more than one file when dealing with multiple environments in one installation.
 */
defined('ENV_FILE') || define('ENV_FILE', '.env');
```